Runtime/ArenaClientScene.cs

```diff
@@ -160,9 +160,9 @@ protected override void Start()
                 }
             }
 
-#if UNITY_EDITOR
+//#if UNITY_EDITOR
```

Comment:
Do we want non-editor builds using the Scene component to automatically connect to the ARENA before they can edit the scene name? Maybe I'm misunderstanding how Runtime mode works. In the Editor builds, you can edit scene name in the Inspector before starting Runtime. In Runtime builds, since you don't have that UI for scene, should we give developers a chance to create their own scene name edit UI before they connect? Maybe there is a better way to do this?

Reply:
I wasn't completely sure how to connect an ARENA scene in build with the #if UNITY_EDITOR tag, so the way I did it was basically spawning a new object with ArenaClientScene.cs attached when I was ready to connect to the scene and then deleting it when disconnecting. I have some companion scripts for setting scenenames in build in a separate script and object. Honestly, it would be best to have a public functions in ArenaClientScene.cs for setting scenenames, namespaces, connecting to a scene, disconnecting to a scene, signin/signout.

Reply:
Yeah, those methods are public here: https://docs.arenaxr.org/content/unity/runtime.html#example. Edward had me write those up a few years back for early Remote Rendering project work with Joey.
```cs
ArenaClientScene scene = ArenaClientScene.Instance;
scene.authType = ArenaMqttClient.Auth.Anonymous;
scene.hostAddress = "arena-dev1.conix.io";
scene.namespaceName = "mwfarb";
scene.sceneName = "example2";
scene.ConnectArena();
scene.DisconnectArena();
```
Thanks for making signout public, I totally overlooked that. Now we can do this as well :-) :
```cs
scene.SignoutArena();
```
Unfortunately, if we have connection occur automatically, it will change the behavior of those apps.

Reply:
Great, in that case, keep it as it is :)

Reply:
At least we can keep the options open for developers that way, adding the ArenaClientScene component to the scene at either design-time or runtime.